controllers/v1/library/categoriesController.js

```
@@ -86,4 +87,109 @@ module.exports = class LibraryCategories extends Abstract {
+				const privateSolutions = await libraryCategoriesHelper.importFromLibrary(
+					req.params._id,
+					req.body,
+					req.query.isATargetedSolution ? req.query.isATargetedSolution : false,
+					req.userDetails
+				)
+
+				return resolve (privateSolutions)
+			} catch (error) {
+				return reject(error)
+			}
+		})
```

Biome (2.1.2)

[error] Promise executor functions should not be `async`.



(lint/suspicious/noAsyncPromiseExecutor)